bin/MadGraph5_aMCatNLO/cards/production/14TeV/Estar_EG_L10000_m1000_to_m6500_pythia8/Estar_EG_L10000_m1000_14TeV_pythia8.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
why are these cfg put under this folder?
they should go under
https://github.com/cms-sw/genproductions/tree/master/python/ThirteenTeV/Estar

Reply:
Dear perrozzi,

I have 2 questions. 
My cfg files are for 14 TeV. Can I put files under Thirteen TeV not FourteenTeV?
And about moving gridpack, can I make directory as below?
/eos/cms/store/group/phys_generator/gridpacks/slc6_amd64_gcc481/14TeV/madgraph/V5_2.4.2/
or do I follow as instruction like /store/group/phys_generator/cvmfs/gridpacks/<your scram architecture>/13TeV/<your generator>/<your version>/ ?
Thank you in advance.

Best regards,
Bobae